Resources/config/consumer.xml

```diff
@@ -17,6 +17,7 @@
             <argument>sonata.media.thumbnail.format</argument>
             <argument type="service" id="sonata.media.thumbnail.format"/>
             <argument type="service" id="sonata.notification.backend" />
+            <argument type="service" id="event_dispatcher" />
```

Comment:
@rande rabbitmq support is really not working without this.. can you please give tell me if i should use a `CompilerPass`